src/sentry/conf/server.py

```diff
@@ -1351,7 +1367,8 @@ def create_partitioned_queues(name):
         "environment": {
             "KAFKA_ZOOKEEPER_CONNECT": "{containers[zookeeper][name]}:2181",
             "KAFKA_LISTENERS": "INTERNAL://0.0.0.0:9093,EXTERNAL://0.0.0.0:9092",
-            "KAFKA_ADVERTISED_LISTENERS": "INTERNAL://{containers[kafka][name]}:9093,EXTERNAL://{containers[kafka][ports][9092/tcp][0]}:{containers[kafka][ports][9092/tcp][1]}",
+            "KAFKA_ADVERTISED_LISTENERS": "INTERNAL://{containers[kafka][name]}:9093,EXTERNAL://{containers[kafka]"
```

Comment:
these were manually formatted by me ( auto formatting was wrapping at a random place and it looked ugly)

Reply:
Got it, thanks. Provided that black reformats this, this change is fine. Otherwise, it would be better not to touch these lines as this makes tracking changes harder.